documentation/ipi-install/modules/ipi-install-node-requirements.adoc

```diff
@@ -3,40 +3,39 @@
 // * installing/installing_bare_metal_ipi/ipi-install-prerequisites.adoc
 
 [id='node-requirements_{context}']
+
 = Node requirements
 
 IPI installation involves a number of hardware node requirements:
 
-- **CPU architecture:** All nodes *must* use `x86_64` CPU architecture.
+- **CPU architecture:** All nodes must use `x86_64` CPU architecture.
 
-- **Similar nodes:** Nodes *should* have an identical configuration per role. That is, control plane nodes *should* be the same brand and model with the same CPU, RAM and storage configuration. Worker nodes should be *identical*.
+- **Similar nodes:** Nodes must have an identical configuration per role. That is, control plane nodes must be the same brand and model with the same CPU, RAM and storage configuration. Worker nodes must be identical.
```

Comment:
We can revisit, I'm not sure its a must.

Reply:
This is also a stylistic issue. The editing team looks at words like "should" as a weasel word. How would you phrase it without using "should"?

Reply:
@johnwilkins @iranzo how about we just say "Nodes are recommended..."

Reply:
Changed to:


- **Similar nodes:** Nodes are recommended to have an identical configuration per role. That is, control plane nodes must be the same brand and model with the same CPU, RAM and storage configuration. Worker nodes must be identical.